documentation/modules/accessing-logs-ui.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+[id="accessing-logs-ui_{context}"]
+= Downloading logs from the web console
+
+You can download logs for a completed, failed, or canceled migration plan and for migrated virtual machines (VMs) by using the {project-short} web console.
+
+.Procedure
+
+. In the web console, click *Migration plans*.
+. To download logs for a migration plan, click *Get logs* beside the migration plan name.
+. To download logs for a migrated VM, click the migration plan name and then click *Get logs* beside the VM.
++
+You can download the VM logs while the migration plan is running.
```

Comment:
The `Get logs` button beside the VM is disabled while the plan is running, and it has a tooltip that says:
"cannot run must gather until the migration is finished"

Reply:
OK. In the mockups, it says that you can download a VM log while the plan is running. I'll look into this.